.gitlab-ci.yml

```diff
@@ -27,6 +28,10 @@ before_script:
   - if [[ -f "environment.$CI_COMMIT_REF_NAME" ]]; then
   -   source environment.$CI_COMMIT_REF_NAME
   - fi
+  - export ADMIN_USER_EMAILS=test@environment.com
+  - aws configure set aws_access_key_id $AWS_ACCESS_KEY_ID
```

Comment:
what access key is this? is this for a service account that you've created?

Reply:
Yep.  `data-store-gitlab-runner`.